### PR TITLE
Added Bundle Entity schemas

### LINKED ISSUE #48: NewSighting schema is missing type key
~~~clojure
(s/defschema NewSighting
  (st/merge
   Sighting
   c/NewBaseEntity
   (st/optional-keys
    {:count s/Int
     :confidence v/HighMedLow})))
~~~

as a result generative tests fail.

### PR DESCRIPTION
**Pull request by gbuisson**
This adds support for a new `Bundle`Entity
includes schemas and generators.
closes https://github.com/threatgrid/iroh-enrich/issues/1
closes #48

### CODE REVIEW

**Comment by gbuisson** [comment]
Do we need CTIA support for this entity?

File: src/ctim/schemas/package.cljc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    :exploit-target_refs ReferenceList
+    :feedback_refs ReferenceList
+    :incident_refs ReferenceList
+    :judgement_refs ReferenceList
+    :sighting_refs ReferenceList
+    :ttp_refs ReferenceList
+    :verdict_refs ReferenceList}))
+
+;; Package already refers to java.lang.Package
+#?(:clj (ns-unmap *ns* 'Package))
```
Comment (gbuisson) [inline src/ctim/schemas/package.cljc]: maybe we should name it to something else and avoid that workaround?

**Comment by yogsototh** [comment]
I would rebase all these commit, otherwise LGTM!

**Comment by gbuisson** [comment]
squashed most commits

**Comment by AlexBaranosky** [comment]
@gbuisson there are still 6 commits. Weren't there 7 originally?

**Comment by gbuisson** [comment]
Yes, the thing is those are not mine and come from other merges, I don't want to rewrite history for them because theyre unrelated to this PR/already on master

**Comment by AlexBaranosky** [comment]
@gbuisson if they are already on master, why can you not `git rebase master` and then your PR won't have merge commit in it?

My workflow is like this:
1) git checkout master
2) git pull mothership master
3) git checkout -b issue-999
4) commit
5) commit
6) git push origin issue-999
7) git checkout master
8) git pull mothership master ## uh oh new changes from master
9) git checkout issue-999
10) git rebase master
11) git push origin issue-999 --force

**Comment by AlexBaranosky** [comment]
Looking nice and squashed 👍

**Comment by gbuisson** [comment]
yeah let's hold on on this, the openc2 changes break testing on CTIA, looking to fix that.

**Pull request merged**